code/__HELPERS/mobs.dm

```diff
@@ -298,7 +298,7 @@ This is always put in the attack log.
 			loglevel = ATKLOG_FEW
 		else if(istype(user) && !user.ckey && !target.ckey) // Attacks between NPCs are only shown to admins with ATKLOG_ALL
 			loglevel = ATKLOG_ALL
-		else if(!user.ckey || !target.ckey) // Player v NPC combat is de-prioritized.
+		else if(!user.ckey || !target.ckey || (user.ckey == target.ckey)) // Player v NPC combat is de-prioritized. Also no self-harm, nobody cares

```

Comment:
If only it would also hide self harm for surrounding players. Oh how we can dream

Reply:
did I hear make hearing self harm messages a preference toggle?

Reply:
Oh baby you just know what I like to hear (and what not)

Reply:
then I know you'll hate to hear that'll only be a thing when you code it 😉